scripts/database/upgrades/upgrade_v4.10.1_to_4.11.sql

```diff
@@ -1,2 +1,6 @@
 ALTER TABLE datasetversion ADD COLUMN archivalcopylocation text;
 ALTER TABLE externaltool ADD COLUMN contenttype text NOT NULL default 'text/tab-separated-values';
+
+ALTER TABLE datavariable
+DROP COLUMN  if exists universe,
+DROP COLUMN  if exists weighted;

```

Comment:
These two columns ("universe" and "weighted") are being dropped? Are they always empty?

Later I noticed that the pull request says "Universe and weighted fields are removed from datavariable, since they are empty in the present dataverse and they are moved to VariableMetadata." So that answers my question. A comment in the script would be nice. Not everyone will see the pull request.